staging/src/k8s.io/apiserver/pkg/endpoints/handlers/response.go

```diff
@@ -49,18 +49,18 @@ func transformObject(ctx context.Context, obj runtime.Object, opts interface{},
 	case target == nil:
 		return obj, nil
 
-	case target.Kind == "PartialObjectMetadata" && target.GroupVersion() == metav1beta1.SchemeGroupVersion:
-		return asV1Beta1PartialObjectMetadata(obj)
+	case target.Kind == "PartialObjectMetadata":
+		return asPartialObjectMetadata(obj, target.GroupVersion())
 
-	case target.Kind == "PartialObjectMetadataList" && target.GroupVersion() == metav1beta1.SchemeGroupVersion:
-		return asV1Beta1PartialObjectMetadataList(obj)
+	case target.Kind == "PartialObjectMetadataList":
+		return asPartialObjectMetadataList(obj, target.GroupVersion())
 
-	case target.Kind == "Table" && target.GroupVersion() == metav1beta1.SchemeGroupVersion:
+	case target.Kind == "Table":
 		options, ok := opts.(*metav1beta1.TableOptions)
```

Comment:
I'm surprised that we'd be still be passing v1beta1 options around?

Reply:
they're aliased the same internally, as mentioned above I wanted to do a separate followup refactor that changes etcd / storage signature (ConvertToTable) globally, and didn't want it blocking the enablement.